sentry_sdk/integrations/_wsgi_common.py

```diff
@@ -124,9 +123,12 @@ def parsed_body(self):
         form = self.form()
         files = self.files()
         if form or files:
-            data = dict(iteritems(form))
-            for key, _ in iteritems(files):
-                data[key] = AnnotatedValue.removed_because_raw_data()
+            data = {}
+            if form:
+                data = dict(form.items())
+            if files:
+                for key in files.keys():
+                    data[key] = AnnotatedValue.removed_because_raw_data()
 
             return data
```

Comment:
It appears to me that if `form or files` evaluates to `False`, then `data` will be undefined, and we will get an error here. If you apply my suggestion from above and also remove the `if form or files` condition (unindenting by one level the lines within the `if` block), this error should be fixed, and the function should otherwise retain the same behavior

Reply:
Not sure I follow -- `if form or files` is `False`, then we don't reach this `return data` line at all since that's only part of the `if` branch, but rather fall back to the `return self.json()`.

Reply:
You're right – because your inline comment was breaking up the `if` block, it looked like the `return data` line was outside the `if` block in the GitHub diff viewer. The `return self.json()` line was also automatically collapsed, which made it seem like `return data` was the only return statement.

Apologies for the confusion!